tests/test_e2e_40_sdntrace.py

```diff
@@ -772,3 +772,61 @@ def test_060_run_sdntrace_order(cls):
         data = response.json()
         list_results = data["result"]
         assert list_results[0][0]["out"]["port"] == 1
+
+    def test_070_run_sdntrace_untagged_vlan(cls):
+        """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
+
+        cls.create_evc("untagged")
+        payload = [
+                    {
+                        "trace": {
+                            "switch": {
+                                "dpid": "00:00:00:00:00:00:00:01",
+                                "in_port": 1
+                            },
+                            "eth": {
```

Comment:
Gretel, let's remove the the `eth` headers with `dl_vlan` for untagged. A packet input shouldn't have a `dl_vlan: 0`. Can you fix this part here for completeness and to avoid confusion, specially also when the rest of API spec validation is in place this would fail.

Reply:
You are right @viniarck. Thanks for pointing it out. Fixed!